### ISSUE TITLE
Error when I try to add a non-existing user to a list

### ISSUE THREAD

**Issue by loicparent** [sendinblue/APIv3-php-library#191]
Hello,

I have a few questions about this API.
I arrive to create or update a user. I arrive also to add it to a specific list.

However, I encounter some issue when I try to add an existing user ou when I try to add a non-existing user to a list.
In fact, I wonder what is the best approach to manage the contacts to avoid having a critical error.

**First question**:
Is is possible de check if the user exists before trying to add it in a list? Or, is it possible to get error in the return of the API instead of having a critical error on the execution?

**Second question**:
Is it possible to remove a user from a specific list rather than delete user from the contact list?

Thanks for your help and have a nice day :)
_Loïc_

**Comment by rajatsib** [comment]
Hi @loicparent 

1. To check if a user exists , you can use [getContactInfo](https://github.com/sendinblue/APIv3-php-library/blob/master/docs/Api/ContactsApi.md#getcontactinfo)

2. To remove a user from a specific list, please use [removeContactFromList](https://github.com/sendinblue/APIv3-php-library/blob/master/docs/Api/ContactsApi.md#removecontactfromlist)

Thanks

**Comment by loicparent** [comment]
Hi @rajatsib,

Thanks for your reply and sorry for my late reaction!

I found these methods but when I tried to call the `getContactInfo()` with a non-existing contact email, I have a fatal error.
I changed a bit my code to use more `Try{} Catch(){}` to avoid having this fatal error. I though it was a bit less invasive such as, for example, a status return instead of an error that kill my script but anyway, I have adapted my code.

_Thanks and have a nice week-end,_
_Loïc_
